common/changes/@microsoft/rush/will-subspace-patch-6_2024-03-04-18-34.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add support for subspace level scoped pnpm-config.jsons",
```

Comment:
Give an example of where this config file would exist.

Reply:
Added